model_compression_toolkit/core/common/graph/base_graph.py

```diff
@@ -33,8 +33,13 @@
 from model_compression_toolkit.logger import Logger
 from model_compression_toolkit.target_platform_capabilities.target_platform.targetplatform2framework import TargetPlatformCapabilities
 
-OutTensor = namedtuple('OutTensor', 'node node_out_index')
-
+# nodes contains all input nodes contributing to outputs of the model
+# output_order contains order (list, dict, or simple node)
+# OutTensor = namedtuple('OutTensor', 'node output_order')
+@dataclass
+class OutTensor:
+    node:BaseNode
```

Comment:
Please add spacing between statements

Reply:
None particular formatting, It's just cleaner and more easy on the eye when there is a proper spacing between statements in my opinion